packages/mux-player/src/index.ts

```diff
@@ -1169,6 +1172,24 @@ class MuxPlayerElement extends VideoApiElement implements MuxPlayerElement {
     this.setAttribute(MuxVideoAttributes.ENV_KEY, `${val}`);
   }
 
+  /**
+   * Get no-volume-pref flag.
+   */
+  get noVolumePref() {
+    return this.hasAttribute(PlayerAttributes.NO_VOLUME_PREF);
+  }
+
+  /**
+   * Set video engine debug flag.
+   */
+  set noVolumePref(val) {
+    if (val) {
+      this.setAttribute(PlayerAttributes.NO_VOLUME_PREF, '');
+    } else {
+      this.removeAttribute(PlayerAttributes.NO_VOLUME_PREF);
+    }
```

Comment:
nitpick(non-blocking): We're moving in the direction of preferring `toggleAttribute()` for these cases, e.g.

```suggestion
this.toggleAttribute(PlayerAttributes.NO_VOLUME_PREF, !!val);
```
Can also add a "changed check" to avoid unnecessary re-rendering, e.g.:
```ts
const boolVal = !!val;
if (boolVal === this.noVolumePref) return;
this.toggleAttribute(PlayerAttributes.NO_VOLUME_PREF, boolVal);
```

Reply:
Feels a bit weird to be the odd one out.